editor/components/post-publish-panel/test/toggle.js

```diff
@@ -57,5 +57,26 @@ describe( 'PostPublishPanelToggle', () => {
 
 			expect( wrapper.prop( 'disabled' ) ).toBe( false );
 		} );
+
+		it( 'should display Schedule… if able to be scheduled', () => {
+			const wrapper = shallow(
+				<PostPublishPanelToggle isPublishable isSaveable isBeingScheduled />
+			);
+			expect( wrapper.childAt( 0 ).text() ).toBe( 'Schedule…' );
+		} );
+
+		it( 'should display Schedule… if able to be published', () => {
```

Comment:
@danielbachhuber Just asking, I'm not sure to be honest, shouldn't it be `should display Publish… if able to be scheduled` instead of `should display Schedule… if able to be published`?

Let me know if it's not correct and if I should fix it.

Reply:
Hm. I think this should be:

```
should display Publish… if able to be published
```

Reply:
I assumed so, let me fix it. Thank you for looking <3

Reply:
Just did in [this commit](https://github.com/WordPress/gutenberg/pull/6729/commits/a872f057b4e4231b8b2a28e069793c0398c59b79).